src/app/meds-n-labels/activity-1/activity-1.component.ts

```diff
@@ -22,7 +23,7 @@ export class MatchmedsComponent implements OnInit {
     canvasTop: number;
     canvasLeft: number;
 
-    height = 320;
+    height = 600;
```

Comment:
If these do not change, then these can be made constants

Reply:
No, they are changing in the application as per the window size.